src/components/Fund/FundDetails/FundDetailsContainer.js

```diff
@@ -161,6 +163,17 @@ const FundDetailsContainer = ({
     [],
   );
 
+  const addBudgetButton = useCallback((status, count) => {
+    return !count && (
+      <Button
+        data-test-add-budget-button
+        onClick={() => toggleAddBudget(status)}
```

Comment:
status? your state is boolean, plus we have a hook for modal toggle

Reply:
Changed name for variable